### PR TITLE
WIP: Add different volume types for the test run

### PR DESCRIPTION
**Pull request by sac-urs**

### CODE REVIEW

File: playbooks/vars/distribute_8bricks.yml [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    lvsize: 200G
+# Mount the devices
+gluster_infra_mount_devices:
+  - { path: '/dist1', vgname: vg_sdb, lvname: gluster_lv_disk1 }
+  - { path: '/dist2', vgname: vg_sdb, lvname: gluster_lv_disk1 }
+  - { path: '/dist3', vgname: vg_sdb, lvname: gluster_lv_disk1 }
+  - { path: '/dist4', vgname: vg_sdb, lvname: gluster_lv_disk1 }
+  - { path: '/dist5', vgname: vg_sdb, lvname: gluster_lv_disk1 }
+  - { path: '/dist6', vgname: vg_sdb, lvname: gluster_lv_disk1 }
+gluster_infra_fw_ports:
```
Comment (amarts) [inline playbooks/vars/distribute_8bricks.yml]: This is common for all the roles. Can it move to something common?

File: playbooks/vars/distribute_8bricks.yml [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+# check which refspec you need to fetch. If you want to test particular
+# patch, you would need to set it up. Remember to uncomment above variable
+# too, if you want to use this. (Visible when you press 'Download'
+# in https://review.gluster.org, particular patch.
+#
+# glusterfs_perf_git_refspec: refs/changes/36/20636/21
+
+#glusterfs_perf_resdir: /var/tmp/glusterperf/{{ansible_date_time.date}}-{{ glusterfs_perf_git_version | default('master') }}
+glusterfs_perf_resdir: "{{ lookup('pipe','date +%Y-%m-%d-%H-%M-%S') }}"
```
Comment (amarts) [inline playbooks/vars/distribute_8bricks.yml]: Prefer it to be common for 1 run. Ie, one result-dir, in which different configs can have different directories to output files.

File: playbooks/vars/replicate_1x3_3servers.yml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+# patch, you would need to set it up. Remember to uncomment above variable
+# too, if you want to use this. (Visible when you press 'Download'
+# in https://review.gluster.org, particular patch.
+#
+# glusterfs_perf_git_refspec: refs/changes/36/20636/21
+
+#glusterfs_perf_resdir: /var/tmp/glusterperf/{{ansible_date_time.date}}-{{ glusterfs_perf_git_version | default('master') }}
+glusterfs_perf_resdir: "{{ lookup('pipe','date +%Y-%m-%d-%H-%M-%S') }}"
+glusterfs_perf_volume: perfvol
+glusterfs_perf_bricks: /gluster_bricks/perfbrick
```
Comment (amarts) [inline playbooks/vars/replicate_1x3_3servers.yml]: is 1 brick enough? I see 6 bricks in above playbook.

Comment (sac-urs) [reply playbooks/vars/replicate_1x3_3servers.yml]: This is one brick per machine. 3 servers.

File: playbooks/vars/distribute_8bricks.yml (hunk 1 above)
Comment (sac-urs) [reply playbooks/vars/distribute_8bricks.yml]: Yes we can do that. And vars_files section will have another entry to include firewall roles.

File: playbooks/vars/distribute_8bricks.yml (hunk 2 above)
Comment (sac-urs) [reply playbooks/vars/distribute_8bricks.yml]: Ack. I haven't thought about perf directory or the organization and structuring results yet.

**Pull request closed without merging**